packages/gafl-webapp-service/assets/sass/rod-licensing/_general.scss

```diff
@@ -113,3 +113,19 @@ div.initially-hidden {
 .phone-number {
   white-space: nowrap;
 }
+
+@media (max-width: 640px) {
+  #continue {
+    position: absolute;
+    top: auto;
+    bottom: 0;
+    left: 15px;
+    right: 15px;
+    width: auto;
+  }
+
+  .govuk-grid-row:has(.payment-summary-wrap) {
+    position: relative;
+    padding-bottom: 75px;
+  }
```

Comment:
I think you've misunderstood my last comment - the #continue button styles need to go inside a parent selector too, otherwise every single continue button will have the same style and so be absolutely positioned, which I don't think is correct.